specification/deviceprovisioningservices/resource-manager/Microsoft.Devices/stable/2020-03-01/iotdps.json

```diff
@@ -2440,6 +2454,10 @@
               "type": "boolean",
               "description": "Indicate if the certificate is verified by owner of private key."
             },
+            "certificate": {
+              "description": "base-64 representation of X509 certificate .cer file or just .pem file content.",
+              "type": "string"
```

Comment:
If this is always base64-encoded, need to add ```"format": "byte"```

Reply:
Its a .cer or pem content. Added a fix.